sdk/python/packages/flet-core/src/flet_core/utils.py

```diff
@@ -159,3 +161,15 @@ def with_degrees(self, value):
         radians = math.radians(value)
         magnitude = abs(self)
         return type(self).polar(radians, magnitude)
+
+
+class DeprecatedEnumMeta(EnumMeta):
+
```

Comment:
I would put that class next to `ThemeVisualDensity` (in the same `[src/flet_core/theme.py](https://github.com/flet-dev/flet/pull/2259/files#diff-24599f89970071ace465e406200accb4cdd5a8be6c55fe1a1f1e25618db7516c)` file) and call it `ThemeVisualDensityDeprecated`. What do you think?

Reply:
Leaving as-is will permit us to use this class on any Enum (add new ifs/cases to the getattribute method), just by passing it as metadata(see VisualDensity). Wdyt?

Reply:
My point is `DeprecatedEnumMeta` is not targeting specific enum, i.e. you are checking just for `item == 'ADAPTIVEPLATFORMDENSITY'` without checking what enum is that. Imagine the switch will grow up for more obsolete enum members.

I suggest having "Deprecated" sibling for every enum that has deprecated members and keep that class next to its enum.

Reply:
Ah, i get your point now! :)
Will give it a look.